core/templates/dev/head/services/AlertsService.js

```diff
@@ -86,18 +86,20 @@ oppia.factory('AlertsService', ['$log', function($log) {
     AlertsService.warnings = [];
   };
 
-  /**
-   * Adds a message, can be info messages or success messages.
-   * @param {string} type - Type of message
-   * @param {string} message - Message content
-   */
-  AlertsService.addMessage = function(type, message) {
+    /**
+     * Adds a message, can be info messages or success messages.
+     * @param {string} type - Type of message
+     * @param {string} message - Message content
+     * @param timeout - Timeout value
```

Comment:
Missing type info. Follow format of the docstrings above this (probably want to say {number|undefined}. Ditto elsewhere.

Also I suggest calling it timeoutMilliseconds (include units). Ditto elsewhere.

Reply:
Done :+1: